README.md

```diff
@@ -121,6 +121,7 @@ Put any of the following options into your `~/.vimrc` in order to overwrite the
 | `let g:bookmark_no_default_key_mappings = 1`                    | 0                        | Prevent any default key mapping from being created|
 | `let g:bookmark_location_list = 1`             | 0                        | Use the location list to show all bookmarks             |
 | `let g:bookmark_disable_ctrlp = 1`             | 0                        | Disable ctrlp interface when  show all bookmarks             |
+| `let  g:bookmark_show_detail_in_ctrlp= 1`             | 1                        | show line number and filename in ctrlp window             |
```

Comment:
Please lets not add another option for this. For simplicity and consistency reasons with the default `location-list` implementation it should always show the file and line number. I believe for the majority of people it's just fine to see `filename` and `line` before the `line content`.